src/oidcendpoint/id_token.py

```diff
@@ -50,7 +52,7 @@ def include_session_id(endpoint_context, client_id, where):
 
 
 def get_sign_and_encrypt_algorithms(
-    endpoint_context, client_info, payload_type, sign=False, encrypt=False
+        endpoint_context, client_info, payload_type, sign=False, encrypt=False
```

Comment:
Fix format

Reply:
That's what PyCharm does when it reformats code. Specifically for method/function arguments.

Reply:
Used black to fix format.